src/js/test/supervisors/FileManager.test.ts

```diff
@@ -275,4 +275,40 @@ describe("FileManager", () => {
       done();
     });
   });
+
+  it(
+    "should remove a coprocessor, if it's removed from active folder and " +
+      "memory, although disable_coproc request fails",
+    function (done) {
+      const handle = createHandle({
+        globalId: hash64(Buffer.from("file"), 0).readBigUInt64LE(),
+      });
+      const {
+        getCoprocessor,
+        moveCoprocessor,
+        disableCoprocessor,
+      } = createStubs(sinonInstance);
+      moveCoprocessor.returns(Promise.resolve(handle));
+      getCoprocessor.returns(Promise.resolve(handle));
+      disableCoprocessor.reset();
+      disableCoprocessor.returns(Promise.reject("error"));
+
+      const repo = new Repository();
+      const removeSpy = sinonInstance.spy(repo, "remove");
+      const file = new FileManager(repo, "submit", "active", "inactive");
+
+      repo.add(handle);
+
+      file.removeHandleFromFilePath(handle.filename, repo).then(() => {
+        assert(removeSpy.called);
+        assert(disableCoprocessor.called);
+        assert.strictEqual(repo.size(), 0);
+        assert(removeSpy.withArgs(handle));
+        disableCoprocessor.firstCall.returnValue.catch(() => {
```

Comment:
You should also add the "negative" case, not just `catch`, and fail the test if the function succeeds.

Reply:
Got it, I'll try

Reply:
Done, I use assert.reject